lib/git_reflow/git_server/pull_request.rb

```diff
@@ -3,6 +3,16 @@ module GitServer
     class PullRequest
       attr_accessor :description, :html_url, :feature_branch_name, :base_branch_name, :build_status, :source_object, :number
 
+      LGTM_REGEX = /(?i-mx:lgtm|looks good to me|[:\\+1:]|:thumbsup:|:shipit:)/
+
+      def self.num_lgtm
+        "#{GitReflow::Config.get('constants.numlgtm')}".length > 0 ? "#{GitReflow::Config.get('constants.numlgtm')}" : ""
```

Comment:
Let's rename this to something more specific, like `approval_minimum` by having the config setting `minimum_approvals` always set, this could be easier to read.
```ruby
def self.approval_minimum
  GitReflow::Config.get('approval_minimum').to_i
end
```